tests/nnfw_api/src/NNPackageTests/SingleSession.test.cc

```diff
@@ -129,3 +129,11 @@ TEST_F(ValidationTestSingleSession, neg_internal_set_config)
   ASSERT_EQ(nnfw_set_config(nullptr, "TRACING_MODE", "0"), NNFW_STATUS_UNEXPECTED_NULL);
   ASSERT_EQ(nnfw_set_config(nullptr, "GRAPH_DOT_DUMP", "0"), NNFW_STATUS_UNEXPECTED_NULL);
 }
+
+TEST_F(ValidationTestSessionCreated, neg_deprecated_api)
+{
+  EXPECT_EQ(nnfw_apply_tensorinfo(nullptr, 0, nnfw_tensorinfo{}), NNFW_STATUS_DEPRECATED_API);
+  EXPECT_EQ(nnfw_prepare_pipeline(nullptr, nullptr), NNFW_STATUS_DEPRECATED_API);
+  EXPECT_EQ(nnfw_push_pipeline_input(nullptr, nullptr, nullptr), NNFW_STATUS_DEPRECATED_API);
+  EXPECT_EQ(nnfw_pop_pipeline_output(nullptr, nullptr), NNFW_STATUS_DEPRECATED_API);
+}
```

Comment:
`nnfw_set_op_backend` may be deprecated according to api description.

Reply:
Right. But I didn't add test for `nnfw_set_op_backend` because internal implementation in compiler(scheduler) is still remained. 
I'll update that after removing internal implementation.